nih-sparc/sparc-app
# Fix issue with project images and sizing

# Description

The purpose of this PR is to fix an issue with project images, and the sizing of the images on all detail pages.

## Type of change


# How Has This Been Tested?

- Go to Find Data
- Click on Projects
- Projects should have an image
- Open a project with an image
- The images should be properly displayed at the top

# Checklist:
